cmd/spire-server/cli/run/run.go

```diff
@@ -353,6 +354,10 @@ func newServerConfig(c *config) (*server.Config, error) {
 		sc.CATTL = ttl
 	}
 
+	if !hasExpectedTTLs(sc.CATTL, sc.SVIDTTL) {
+		sc.Log.Warn("ca_ttl is less than default_svid_ttl * 6. SPIRE Server prepares a new CA certificate when 1/2 of the CA lifetime has elapsed in order to give ample time for the new trust bundle to propagate. However, it does not start using it until 5/6th of the CA lifetime. So its normal for an SVID TTL to be capped to 1/6th of the CA TTL. In order to get the expected lifetime on SVID TTLs, the CA TTL should be 6x.")
```

Comment:
This message is a little long and more detailed than it needs to be I think? I also feel it makes a little more sense to talk about it in terms of the svid_ttl. How about something like this?
`sc.Log.Warn("The configured SVID TTL cannot be guaranteed in all cases - SVIDs with shorter TTLs may be issued if the signing key is expiring soon. Set a CA TTL of at least %v or reduce SVID TTL below %v to avoid issuing SVIDs with a smaller TTL than specified.")`